build.sbt

```diff
@@ -63,6 +63,8 @@ libraryDependencies ++= {
 
 Revolver.settings.settings
 
+Revolver.enableDebugging(port = 5051, suspend = false)
```

Comment:
On production, the system should not be started in this way - so this should not be a problem. The only case where this comes up is if you run re-start/re-stop in the sbt console.